staging/src/k8s.io/csi-translation-lib/plugins/gce_pd.go

```diff
@@ -211,8 +211,9 @@ func (g *gcePersistentDiskCSITranslator) TranslateInTreeInlineVolumeToCSI(volume
 
 	return &v1.PersistentVolume{
 		ObjectMeta: metav1.ObjectMeta{
-			// A.K.A InnerVolumeSpecName required to match for Unmount
-			Name: volume.Name,
+			// Must be unique per disk as it is used as the unique part of the
+			// staging path
+			Name: pdSource.PDName,
```

Comment:
This name may not be globally unique. For example, if you use NFS and GCE PD at the same time, the share name and the PD name may be the same resulting in a collision. We should stick the plugin name in there to avoid that collision.